pkg/resource/webhook_secret.go

```diff
@@ -0,0 +1,196 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
Are we supporting custom certs for this as well?

Reply:
Indeed. If someone would like to use their own certs, they need to create the TLS secret before running the operator. Any idea where the best place to mention that would be? I have a small line in a comment in _prep_webhooks.go_, but might be worth calling it our more prominently.